ibm/service/kubernetes/resource_ibm_container_vpc_cluster.go

```diff
@@ -340,6 +340,14 @@ func ResourceIBMContainerVpcCluster() *schema.Resource {
 				DiffSuppressFunc: flex.ApplyOnce,
 			},
 
+			"disable_outbound_traffic_protection": {
```

Comment:
please add this field to the documentation too
https://github.com/IBM-Cloud/terraform-provider-ibm/blob/master/website/docs/r/container_vpc_cluster.html.markdown